Makefile

```diff
@@ -76,12 +76,15 @@ $(OUTPUT_DIRNAME)/$(DOC_FILENAME).html: $(DOC_FILES) $(FIGURE_FILES)
 	ls -sh $(shell readlink -f $@)
 endif
 
-validate-examples:
+validate-examples: schema/fs.go
```

Comment:
maybe `schema/fs.go` doesn't even need to be commited then ... only generated. It often causes a lot of senseless rebasing.